Pod/Assets/Scripts/iconVersioning.sh

```diff
@@ -43,87 +48,100 @@ shopt -u extglob
 caption="${version} ($build_num)\n${branch}\n${commit}"
 echo $caption
 
-function abspath() { pushd . > /dev/null; if [ -d "$1" ]; then cd "$1"; dirs -l +0; else cd "`dirname \"$1\"`"; cur_dir=`dirs -l +0`; if [ "$cur_dir" == "/" ]; then echo "$cur_dir`basename \"$1\"`"; else echo "$cur_dir/`basename \"$1\"`"; fi; fi; popd > /dev/null; }
+function abspath() {
+	pushd . > /dev/null
+
+	if [ -d "$1" ]; then
+		cd "$1"
+		dirs -l +0
+	else
+		cd "`dirname \"$1\"`"
+		cur_dir=`dirs -l +0`
+
+		if [ "$cur_dir" == "/" ]; then
+			echo "$cur_dir`basename \"$1\"`"
+		else
+			echo "$cur_dir/`basename \"$1\"`"
+		fi
+	fi
+
+	popd > /dev/null
+}
 
 function processIcon() {
     base_file=$1
-    
+
     cd "${CONFIGURATION_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
-    base_path=`find . -name ${base_file}`
-    
+    base_path=`find . -maxdepth 2 -name ${base_file}`
```

Comment:
This can be replaced by:

    base_path=`find . -name ${base_file} -print -quit`

which is less hacky than this `2` depth limit.